src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -80,7 +80,15 @@ protected function executeCommand(array $command_args)
         $command_summary = $this->getCommandSummary($command_args);
         $command_line = $this->getCommandLine($command_args);
 
-        $ssh_data = $this->sendCommandViaSsh($command_line);
+        // Retrieve the trace ID from the container
+        $trace_id = $this->getContainer()->get('trace_id');
```

Comment:
Could we instead factor Request::getTraceId() out into a TraceId class, and just call the static method where needed? `$this->getContainer()` is an anti-pattern; Terminus over-uses it, but we don't need to do it here.

Reply:
OK to skip this request if the adjustment would take too long.